docker/ci/helper.Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+ARG SOURCES_DIR=/usr/src/ipa
+FROM rust:latest as builder
+ARG SOURCES_DIR
+LABEL maintainer="akoshelev"
+
+# Prepare helper binaries
+WORKDIR "$SOURCES_DIR"
+COPY . .
+RUN set -eux; \
+    cargo build --bin helper --release --no-default-features --features "web-app real-world-infra"
+
+# Copy them to the final image
+FROM debian:bullseye-slim
+ENV HELPER_BIN_PATH=/usr/local/bin/ipa-helper
+ENV CONF_DIR=/etc/ipa
```

Comment:
This isn't created here, so I would instead include that in the entrypoint script (or as a command line flag), once that is available.